hoover/search/es.py

```diff
@@ -1,17 +1,34 @@
+import json
+from contextlib import contextmanager
 from django.conf import settings
 from elasticsearch import Elasticsearch
 from elasticsearch.client.utils import _make_path
 from elasticsearch.exceptions import NotFoundError, RequestError, ConnectionError
 
-es = Elasticsearch(settings.HOOVER_ELASTICSEARCH_URL)
 DOCTYPE = 'doc'
 
 class SearchError(Exception):
     def __init__(self, reason):
         self.reason = reason
 
+@contextmanager
+def elasticsearch():
```

Comment:
This context manager is useful enough on its own. Would you mind creating a pull request just with it? I'm also working on code in `es.py` that would be impacted by this patch.